lib/spack/spack/cmd/create.py

```diff
@@ -394,6 +394,25 @@ def __init__(self, name, *args, **kwargs):
         super(SIPPackageTemplate, self).__init__(name, *args, **kwargs)
 
 
+class GoPackageTemplate(PackageTemplate):
+    """Provides appropriate overrides for Go-based packages."""
+
+    base_class_name = 'GoPackage'
+
+    dependencies = """\
+    # FIXME: GoPackage's default might not suit, adjust as necessary
+    # depends_on('go@1.X.Y:', type='build')
+
+    # FIXME: Add additional dependencies if required.
+    # depends_on('foo')"""
+
+    body_def = """\
+    # FIXME: Add any add'l build flags (-mod=vendor is added automagically)
+    # build_flags = []
+    # FIXME: [REQUIRED] Add names of executables to install into prefix.bin
```

Comment:
Add a new line before this

Reply:
err, between the `body_def` and the first line?

Reply:
😕 

...as near as I can see I'm doing exactly what AutoreconfPackageTemplate is doing (standing on the shoulders of giants, etc...).